documentation/course/e2/README.md

```diff
@@ -631,38 +671,69 @@ TODO @tchojnacki
     <th>Wady</th>
   </tr>
   <tr>
-    <th>Rozwiązanie 1</th>
+    <th>Komunikacja API Gateway → Account Service przy każdym zapytaniu</th>
     <td>
       <ul>
-        <li>Zaleta 1</li>
+        <li>Zachowanie SRP</li>
+        <li>Brak dociążenia API Gateway</li>
+        <li>Brak potrzeby dzielenia kluczy publicznych</li>
       </ul>
     </td>
     <td>
       <ul>
-        <li>Wada 1</li>
+        <li>Duże opóźnienie na zapytaniu</li>
+        <li>Dociążenie Account Service</li>
```

Comment:
```suggestion
        <li>Dociążenie Account Service</li>
        <li>Powtarzalność kodu sprawdzania dostępu</li>
```

Reply:
I don't understand, the validation code is never duplicated, it just moves around depending on the solution:
- **Solution 1:** only in Account Service
- **Solution 2:** only in API Gateway
- **Solution 3:** only in the sidecar

Reply:
I have no idea why (maybe because those raw tables are awful to read), but I read it, and somehow different topic  materialized in my head. I thought it read something like "every service checks user permission byitself". That's also why before I mentioned reverse-proxy. My bad.